sample/build.gradle

```diff
@@ -71,11 +71,11 @@ android {
 }
 
 dependencies {
-//    implementation libs.library.retrofit
-//    implementation libs.library.ktor
-    implementation project(":halley-core")
-    implementation project(":halley-retrofit")
-    implementation project(":halley-ktor")
+    implementation libs.library.retrofit
+    implementation libs.library.ktor
+//    implementation project(":halley-core")
+//    implementation project(":halley-retrofit")
+//    implementation project(":halley-ktor")
```

Comment:
I think it is fine to have a reference to local modules rather than release ones in sample app, but ok

Reply:
I thought the sample app should serve as an example of the lib integration as well, so that’s why I commented the setup I used for development testing.